### DISCUSSION TITLE
Fast logout period

### DISCUSSION THREAD

**Discussion by mr2017x** [kimai/kimai#2729] [General]
**Describe the bug**
Users are getting logged out to fast - a session should be at least be active one day (8-16h). It seems as this is just since last or the update before the last.

**To Reproduce**
1. Log in
2. work with the tool
3. e.g. stop time: Error

**Additional context**
 - Kimai version: 1.14.3 (shown as 1.14.1) - was updated, but demo also shows 1.14.1
 - PHP version: 7.4.22
 - Device: Windows Device
 - Browser Chrome

**Comment by kevinpapst** [comment]
Please report back after reading and adjusting: https://www.kimai.org/documentation/users.html#session-lifetime
